### PR TITLE
Add Swagger.json and update snippets

### LINKED ISSUE #398: Update HTTP docs to v3

### PR DESCRIPTION
**Pull request by okdistribute**
Fixes #398 

TOOD
- [ ] swagger.json for v3

### CODE REVIEW

File: docs/http/api.md [hunk 1]
```diff
@@ -5,761 +5,19 @@ sidebar_position: 2
...
-    "externalDocs": {
-      "url": "https://docs.ditto.live/http/installation/",
-      "description": "For more detailed instructions on how to use this API and the Ditto SDK, please see the documentation."
-    }
-}
-  ```
-</details>
+The Open API `swagger.json` spec can be found at the following URL:
 
+* `https://{app_id}.cloud.ditto.live/api/v2/swagger.json`
```
Comment (kw7oe) [inline docs/http/api.md:10]: should we add one for `/api/v3` as well? at least it doesn't confuse the users as the rest are `/v3` while this is `/v2`

**Review by kw7oe — APPROVED** [review]
Otherwise, LGTM 👍🏻

File: docs/http/api.md (hunk 1 above)
Comment (okdistribute) [reply docs/http/api.md:10]: Yes it would be helpful

Comment (kw7oe) [reply docs/http/api.md:10]: let me add the issue first, then i'll work on it after christmas so it will be available on the next release

**Pull request merged**